Netezos/Utils/Lengths.cs

```diff
@@ -0,0 +1,275 @@
+﻿namespace Netezos
+{
```

Comment:
Introduced this `Lengths` class to represent a table I was using for reference.

https://github.com/baking-bad/pytezos/blob/f94160821a10503a1f2d9f5967be7ab9362ec019/pytezos/encoding.py#L9

Reply:
Please, remove unused classes like `id`, or `GSp`, etc.